yml/cdb_latiss.yaml

```diff
@@ -0,0 +1,829 @@
+    datatype: string
+    length: 20
+    nullable: false
+    description: Official name of the exposure.
+  - name: controller
+    "@id": "#Exposure.controller"
+    datatype: string
+    length: 1
+    nullable: false
+    description: The abbreviation of the controller used for the observation (O, C)
+  - name: day_obs
+    "@id": "#Exposure.day_obs"
+    datatype: long
+    nullable: false
+    description: Day of observation
+  - name: seq_num
+    "@id": "#Exposure.seq_num"
+    datatype: long
+    nullable: false
+    description: Sequence number
+  - name: physical_filter
+    "@id": "#Exposure.physical_filter"
+    datatype: string
+    length: 64
+    description: ID of physical filter, the filter associated with a particular instrument.
+  - name: band
+    "@id": "#Exposure.band"
+    datatype: string
+    length: 32
+    description: Name of the band used to take the exposure where this source was
+      measured. Abstract filter that is not associated with a particular instrument.
+  - name: ra
+    "@id": "#Exposure.ra"
+    datatype: double
+    description: RA of targeted focal plane center.
+  - name: decl
+    "@id": "#Exposure.decl"
+    datatype: double
+    description: Declination of targeted focal plane center
+  - name: skyRotation
+    "@id": "#Exposure.skyRotation"
+    datatype: double
+    description: Targeted sky rotation angle.
+  - name: azimuth_start
+    "@id": "#Exposure.azimuth_start"
+    datatype: double
+    description: Azimuth of focal plane center at the start of the exposure.
+  - name: azimuth_end
+    "@id": "#Exposure.azimuth_end"
+    datatype: double
+    description: Azimuth of focal plane center at the end of the exposure.
+  - name: azimuth
+    "@id": "#Exposure.azimuth"
+    datatype: double
+    description: Azimuth of focal plane center at the middle of the exposure.
+  - name: altitude_start
+    "@id": "#Exposure.altitude_start"
+    datatype: double
+    description: Altitude of focal plane center at the start of the exposure.
+  - name: altitude_end
+    "@id": "#Exposure.altitude_end"
+    datatype: double
+    description: Altitude of focal plane center at the end of the exposure.
+  - name: altitude
+    "@id": "#Exposure.altitude"
+    datatype: double
+    description: Altitude of focal plane center at the middle of the exposure.
+  - name: zenithDistance_start
+    "@id": "#Exposure.zenithDistance_start"
+    datatype: double
+    description: Zenith distance at the start of the exposure.
+  - name: zenithDistance_end
+    "@id": "#Exposure.zenithDistance_end"
+    datatype: double
+    description: Zenith distance at the end of the exposure.
+  - name: zenithDistance
+    "@id": "#Exposure.zenithDistance"
+    datatype: double
+    description: Zenith distance at the middle of the exposure.
+  - name: airmass
+    "@id": "#Exposure.airmass"
+    datatype: double
+    description: Airmass of the observed line of sight at the middle of the exposure.
+  - name: expMidpt
```

Comment:
The `length` of the `timestamp` datatype is a problematic concept, as the modifier in the underlying database type is a precision, not a length.  `64` is unlikely to be a useful value in any event.

Reply:
The same comment applies in several other places.

Reply:
Using `postgresql:datatype: TIMESTAMP(6)` might be the best option for now given current limitations in the Felis data model.